README.rst

```diff
@@ -69,13 +69,15 @@ the coverage at least stays the same before you submit a pull request.
 License
 -------
 
-Distributed under the terms of the `GNU GPL v3.0`_ license, "kiwitcms-pytest-plugin" is free and open source software
+Distributed under the terms of the `GNU GPL v3.0`_ license,
+"kiwitcms-pytest-plugin" is free and open source software
```

Comment:
Does README need to be updated ?

Reply:
I'm getting `README.rst:72: D001 Line too long` from Doc8